nano/rpc_test/rpc.cpp

```diff
@@ -6866,3 +6866,18 @@ TEST (rpc, confirmation_info)
 		ASSERT_EQ (0, response.get<unsigned> ("total_tally"));
 	}
 }
+
+TEST (rpc, election_statistics)
+{
+	nano::test::system system;
+	auto node1 = add_ipc_enabled_node (system);
+	auto const rpc_ctx = add_rpc (system, node1);
+	boost::property_tree::ptree request1;
+	request1.put ("action", "election_statistics");
+	auto response1 (wait_response (system, rpc_ctx, request1));
+	ASSERT_EQ ("0", response1.get<std::string> ("normal"));
+	ASSERT_EQ ("0", response1.get<std::string> ("hinted"));
+	ASSERT_EQ ("0", response1.get<std::string> ("optimistic"));
+	ASSERT_EQ ("0", response1.get<std::string> ("total"));
+	ASSERT_EQ ("0.00", response1.get<std::string> ("aec_utilization_percentage"));
```

Comment:
This could at least start a single election which would ensure reported values work as expected

Reply:
@pwojcikdev I'm not sure how to do this. If I start a normal election it might get confirmed before the RPC request is executed.